ara/models.py

```diff
@@ -31,6 +32,30 @@ def mkuuid():
 
     return str(uuid.uuid4())
 
```

Comment:
I don't think this adds much bloat, and I think the bits about relationships help make things much more clear when reading the code.